test/lyapunovs/continuous_lyapunov.jl

```diff
@@ -27,6 +27,10 @@ println("\nTesting continuous system lyapunov exponents...")
   @testset "lyapunov" begin
     λ1 = lyapunov(ds, 10000.0)
     @test 0.89 < λ1 < 0.92
+    λ_ts, tvec = lyapunov_full(ds, 10000.0)
+    @test λ1 == λ_ts[end]
```

Comment:
you should change this to use `approx` instead of `==`. You almost never get the exact same result with 2 different calls to the function

Reply:
So `@test λ1 ≈ λ_ts[end]` should be ok?

Reply:
probably :P We will see when the tests start! We might need to increase the default tolerance of approx (it is 1e-8)